docs/manual/nullness-checker.tex

```diff
@@ -856,10 +856,16 @@ \section{Other tools for nullness checking\label{nullness-related-work}}
 
 If your code is already annotated with a different nullness
```

Comment:
Should we add something like this:
This support is mainly for third-party libraries.  If you use the Checker Framework to check your code, you should use the Checker Framework nullness annotations.

Reply:
This is true, but I don't want to repeat this 4 times in the manual.  I think it's OK to omit.